### ISSUE TITLE
Compat32: ALSA doesn't work

### ISSUE THREAD

**Issue by Unrud** [flatpak/freedesktop-sdk-images#48]
ALSA doesn't work for 32-bit applications with ``org.freedesktop.Platform.Compat32``.

I compiled the [pcm_min](https://www.alsa-project.org/alsa-doc/alsa-lib/_2test_2pcm_min_8c-example.html) example program for 32-bit and get the following error:

```
ALSA lib dlmisc.c:254:(snd1_dlobj_cache_get) Cannot open shared library /usr/lib/alsa-lib/libasound_module_pcm_pulse.so
Playback open error: No such device or address
```

The problem seems to be that the library path is hard-coded. It tries to load ``/usr/lib/alsa-lib/libasound_module_pcm_pulse.so``, instead of searching for the 32-bit library from Compat32.

Related: https://github.com/flathub/com.valvesoftware.Steam/issues/31

**Comment by alexlarsson** [comment]
I wonder if there is an env var to change that...

**Comment by alexlarsson** [comment]
Or maybe we can put something in asound.conf.

**Comment by valentindavid** [comment]
This should be fix in the coming build of abeb88ed6834ea43fbd086103bcb0db39e43222f. We can close this one.

**Comment by Unrud** [comment]
Fixed by #51
